x-pack/plugins/observability/public/pages/alerts/alerts_search_bar.tsx

```diff
@@ -35,7 +35,7 @@ export function AlertsSearchBar({
     <SearchBar
       indexPatterns={dynamicIndexPattern}
       placeholder={i18n.translate('xpack.observability.alerts.searchBarPlaceholder', {
-        defaultMessage: '"domain": "ecommerce" AND ("service.name": "ProductCatalogService" …)',
+        defaultMessage: '"rule.id" : "metrics.alert.threshold"',
       })}
```

Comment:
I was thinking that maybe the rule.id example is not the best placeholder example, since users don't even know the rule ids. I didn't use any of the kibana.alert.rac fields in the placeholder, since there is an open PR that changes the field prefix https://github.com/elastic/kibana/pull/105096

Reply:
Good point. But since we know what the new field names will be I don't see a problem with using them in the placeholder text already.

Reply:
@weltenwort I used the status field

Reply:
The status is already filtered for by the "open"/"closed"/"all" buttons, right? Maybe it's misleading to also suggest this as a common filter in the query bar?

Reply:
@weltenwort Yep good point. I used the threshold and used the new field name`kibana.alert.evaluation.threshold`